api_docs/changelog.md

```diff
@@ -20,6 +20,10 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 190**
+
+* [`DELETE /realm/emoji/{emoji_name}`](/api/deactivate-custom-emoji): Sending `404` status code when emoji does not exist instead of `400`.
```

Comment:
Ths **Changes** entry both here and in `zulip.yaml` do not use the same phrasing style that we use for other API changes. @laurynmm can help if needed but you should be able to fix this by just studying the wording we use elsewhere.